app/src/main/java/net/gsantner/markor/frontend/textview/HighlightingEditor.java

```diff
@@ -95,6 +107,52 @@ public void afterTextChanged(final Editable s) {
         setEmojiCompatEnabled(false);
     }
 
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+        defaultPaddingLeft = getPaddingLeft();
+    }
+
+    @Override
+    protected void onDraw(Canvas canvas) {
```

Comment:
Good thing to test: see how line numbers behave if you
1. disable / enable Line wrapping
2. disable / enable highlighting (especially with bigger headings enabled)
3. Change text size

when line numbers are enabled. These should all trigger a redraw, but it would be good to check

Reply:
Now, it all work well with these options.